plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -313,14 +316,19 @@ fu_plugin_thunderbolt_add (FuPlugin *plugin, GUdevDevice *device)
 		return;
 	}
 
-	vid = fu_plugin_thunderbolt_udev_get_id (device, "vendor", &error_vid);
+	vid = fu_plugin_thunderbolt_udev_get_guint16 (device, "vendor", &error_vid);
```

Comment:
I think fu_plugin_thunderbolt_udev_get_uint16 is more in keeping with the other code.